.ci_support/osx_arm64_.yaml

```diff
@@ -22,8 +22,6 @@ cxx_compiler_version:
 - '16'
 glib:
 - '2'
-gstreamer:
-- '1.24'
```

Comment:
Pretty sure that the selectors using the version are at fault here:
```
    - gstreamer                          # [unix and version == "6.7.1"]
    - pulseaudio                         # [linux and version == "6.7.1"]
```

This feedstock only builds one version at a time. Why are we trying to filter dependencies on the version at all?

In any case, if you want to keep that filter, maybe you'll have more success with:
```
    {% if version == "6.7.1" %}
    - gstreamer                          # [unix]
    - pulseaudio                         # [linux]
    {% endif %}
```

Reply:
I'll try rerendering here but, maybe it's because of the version check with:
```
    - gst-plugins-base  {{ gstreamer }}  # [unix and version == "6.7.1"]
    # gstreamer plugin not supported on win32: https://bugreports.qt.io/browse/QTBUG-107073
    - gstreamer                          # [unix and version == "6.7.1"]
```

Is there a way to pin this with a conditional on version?